tests/release_tests/test_launch/launch-agent.yml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
```

Comment:
Ideally we'd use the manifests in wandb/sdk/launch/deploys/kubernetes or the helm chart at https://github.com/wandb/helm-charts/tree/main/charts/launch-agent rather than spinning up a new version